web/components/og/graph.tsx

```diff
@@ -81,3 +79,39 @@ const computeColorStops = (
   }
   return stops
 }
+
+export function Sparkline(props: {
+  data: ogPoint[]
+  height: number
+  /** width / height */
+  aspectRatio?: number
+  min: number
+  max: number
+}) {
+  const { data, height: h, aspectRatio = 1, min, max } = props
+  const w = h * aspectRatio
+  const visibleRange = [data[0].x, data[data.length - 1].x]
+  const curve = curveStepBefore
+
+  const xScale = scaleTime(visibleRange, [4, w - 4])
```

Comment:
This seems like a pretty weird way to make a margin, why not make the `scaleTime` range 0 to width and then use CSS to make a margin?

Reply:
purpose was to leave enough room in svg for the final points not to be cut off (since the line is 4px thick)